contracts/flows/PolicyDefaultFlow.sol

```diff
@@ -41,15 +41,27 @@ contract PolicyDefaultFlow is
         PolicyController policy = getPolicyContract();
         require(
             policy.getPolicy(processId).state == IPolicy.PolicyState.Expired,
-            "ERROR:PFD-001:POLICY_NOT_EXPIRED"
+            "ERROR:PFD-002:POLICY_NOT_EXPIRED"
         );
         _;
     }
 
+    modifier onlyResponsibleProduct(bytes32 processId) {
+        PolicyController policy = getPolicyContract();
+        IPolicy.Metadata memory metadata = policy.getMetadata(processId);
+        ComponentController component = ComponentController(getContractFromRegistry("Component"));
+        require(metadata.productId == component.getComponentId(address(msg.sender)), "ERROR:PFD-003:PRODUCTID_BELONGS_TO_OTHER_PRODUCT");
```

Comment:
change ERROR:PFD-003:PRODUCTID_BELONGS_TO_OTHER_PRODUCT to ERROR:PFD-003:PROCESSID_PRODUCT_MISMATCH